packages/hash/frontend/src/lib/entities.ts

```diff
@@ -444,20 +450,20 @@ export const convertApiEntityTypesToBpEntityTypes = (
  * @see https://blockprotocol.org/docs/spec/graph-service-specification#json-schema-extensions
  */
 export const generateEntityLabel = (
-  rootEntityAndSubgraph:
-    | RootEntityAndSubgraph
+  entitySubgraph:
+    | Subgraph<SubgraphRootTypes["entity"]>
     | Partial<{ entityId: string; properties: any }>,
```

Comment:
Why is this alternative type allowed if we check for `(!("roots" in entitySubgraph))` and throw otherwise below?

Reply:
Frontend breaks in older code if we don't leave this in for now :( (entityMenu stuff for example)

Reply:
I mean it's not different from main so I'd like to not change it either way. But I think so, yes. It's existing broken parts of the app though like the block context menu